app/main/views/search.py

```diff
@@ -59,6 +59,20 @@ def create(index_name):
 
 @main.route('/<string:index_name>', methods=['DELETE'])
 def delete(index_name):
+    status, status_code = status_for_index(index_name)
+
+    if status_code != 200:
```

Comment:
I couldn't find any documentation either, which is why I asked the question! Happy to put it in for now and then revise in future if it becomes an issue.